### ISSUE TITLE
Questions about saturation and ConcentrationDescriber.

### ISSUE THREAD

**Issue by zepumph** [phetsims/molarity#176]
* In ConcentrationDescriber.solidsToIndex there is this comment: 
  ```js
    // there is an unnamed region right after saturation point that doesn't have a description, so 1 more region must be
    // added to the length of SOLIDS_STRINGS.
    const numberOfIncrements = SOLIDS_STRINGS.length + 1;
  ```
  This seems very confusing to me. Why is this unnamed region defined here instead of in the regions list? Why does it take up the same amount of range as all others? ". . .right after saturation point. . . "-- how is this defined, as it seems like the unnamed region isn't tied to any particular saturation value? 

* MolarityModel has a `isSaturated` function that is based solely on how much precipitate there is. I think I recall hearing something recently that the a11y team has discovered this may not be as closely tied, as it is possible for it to be saturated without yet creating any precipitate. Let's talk to see if the model needs to be reworked, as it seems like the model's `isSaturated` function should be the ground truth in this case.

**Comment by twant** [comment]
I think the second bullet is addressed in #154. 

The first bullet I believe has been fixed when I refactored the toIndex functions in #173 (see 22ae634bb7c09c9bf950846fc40b8e0c68a46d43). @zepumph assigning over to you to review.

**Comment by zepumph** [comment]
Looks great, see you in that other issue. Closing
